### PR TITLE
Fix get nameservers

### LINKED ISSUE #2997: netcfg/get_nameservers in the boot prompt is split and joined with commas after each character
### Steps to reproduce

1. for instance use `--kopts` on Ubuntu/Debian to set the DNS as follows: `netcfg/get_nameservers=1.9.168.0.1`
2. the boot prompt will contain: `netcfg/get_nameservers=1,.,9,.,1,6,8,.,0,.,1`
3. even if I try to use and array, it always interpolated as a string and I get: `netcfg/get_nameservers=[,",1,.,9,.,1,6,8,.,0,.,1,",]`

### Expected behavior

the boot prompt should contain `netcfg/get_nameservers=1.9.168.0.1`

### Cobbler version

````
4.8.5
````

### Operating system

CentOS 7

### Cobbler log

N/A

### Screenshots

N/A

### Additional information

I have PR ready

### PR DESCRIPTION
**Pull request by maxadamo**
resolves #2997 and it's referring to version 2.8.5

### CODE REVIEW

**Comment by SchoolGuy** [comment]
@maxadamo Could you please sign your commit if you have a GPG key? And if possible also try to rebase this commit against `main`? Or is this error just occurring with 2.8.5?

File: cobbler/action_buildiso.py [hunk 1]
```diff
@@ -397,7 +397,10 @@ def generate_netboot_iso(self,imagesdir,isolinuxdir,profiles=None,systems=None,e
                    else:
                       append_line += " dns=%s" % my_dns
                 if dist.breed in ["ubuntu","debian"]:
-                   append_line += " netcfg/get_nameservers=%s" % ",".join(my_dns)
+                   if type(my_dns) == list:
```
Comment (SchoolGuy) [inline cobbler/action_buildiso.py:400]: I believe the following suggestion is the recommended way to do type checks in Python. Your check will only work with an _exact_ list. The proposed change would also work with an object which is a subclass of a `list`.

```suggestion
                   if isinstance(my_dns, list):
```

Comment (maxadamo) [reply cobbler/action_buildiso.py:400]: @SchoolGuy I certainly agree, with this comment but, if you check the code, I used the same style which was already used in the script. If we change this one we need to change the other occurrences.

**Comment by maxadamo** [comment]
> @maxadamo Could you please sign your commit if you have a GPG key? And if possible also try to rebase this commit against `main`? Or is this error just occurring with 2.8.5?

The main branch does not have the file: `action_buildiso.py` 

p.s.: I didn't notice that I have lost my GPG signature

File: cobbler/action_buildiso.py (hunk 1 above)
Comment (SchoolGuy) [reply cobbler/action_buildiso.py:400]: Ah okay. Then let's not change the pattern.

**Comment by SchoolGuy** [comment]
@maxadamo The equivalent on `main` is this:

https://github.com/cobbler/cobbler/blob/9bebebd3b3a8fe9ec8bad99f49856b9672ce2b03/cobbler/actions/buildiso/netboot.py#L90-L110

**Comment by maxadamo** [comment]
the GPG signature now looks good. 
Do you want me to change the other occurrences to make use of the other syntax? 
`if isinstance(var, list):`

**Review by SchoolGuy — APPROVED** [review]
@maxadamo No I am fine with the state of this PR. We can merge it later. I need to have a detailed look at all backports on the weekend finally. (Wanted to this initially last weekend already)

**Pull request merged**